### PR TITLE
Add dictionary parameter to decompress method

### PR DESCRIPTION
**Pull request by bra-fsn**
Fixes #45

### CODE REVIEW

File: src/brotli/brotli.py [hunk 1]
```diff
@@ -75,13 +75,14 @@ class BrotliEncoderMode(enum.IntEnum):
 MODE_FONT = BrotliEncoderMode.FONT
 
 
-def decompress(data):
+def decompress(data,dictionary=b''):
```
Comment (Lukasa) [inline src/brotli/brotli.py]: Can you put a space after this comma please?

**Comment by alexwlchan** [comment]
I think this is actually associated with #57.

**Comment by KOLANICH** [comment]
damn, almost 3 years have passed.

**Comment by sethmlarson** [comment]
@KOLANICH You're welcome to pick up where this PR has left off if this is a feature you really need.

**Pull request closed without merging**